docs/book/src/reference/jobs.md

```diff
@@ -9,26 +9,37 @@ It also documents the cluster-api specific configuration in test-infra.
 
 ### Presubmits
 
-**Legend**:
-* ✳️️ jobs that don't have to be run successfully for merge
-* ✴️ jobs that are not triggered automatically for every commit
-
 Prow Presubmits:
-* [pull-cluster-api-build-main] `./scripts/ci-build.sh`
-* ✳️️ [pull-cluster-api-apidiff-main] `./scripts/ci-apidiff.sh`
-* [pull-cluster-api-verify-main] `./scripts/ci-verify.sh`
-* [pull-cluster-api-test-main] `./scripts/ci-test.sh`
-* [pull-cluster-api-test-mink8s-main] `./scripts/ci-test.sh`
-* [pull-cluster-api-e2e-main] `./scripts/ci-e2e.sh`
-  * GINKGO_FOCUS: `[PR-Blocking]`
-* ✳️️ [pull-cluster-api-e2e-informing-main] `./scripts/ci-e2e.sh`
-  * GINKGO_FOCUS: `[PR-Informing]`, GINKGO_SKIP: `[IPv6]`
-* ✳️️ [pull-cluster-api-e2e-informing-ipv6-main] `./scripts/ci-e2e.sh`
-  * GINKGO_FOCUS: `[IPv6] [PR-Informing]`, IP_FAMILY: `IPv6`
-* ✳️️ ✴️ [pull-cluster-api-e2e-full-main] `./scripts/ci-e2e.sh`
-  * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade] [IPv6]` (i.e. "no tags")
-* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-27-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.27` TO: `ci/latest-1.28`
-  * GINKGO_FOCUS: `[K8s-Upgrade]`
+* mandatory for merge, always run:
+  * [pull-cluster-api-build-main] `./scripts/ci-build.sh`
+  * [pull-cluster-api-verify-main] `./scripts/ci-verify.sh`
+* mandatory for merge, run if go code changes:
+  * [pull-cluster-api-test-main] `./scripts/ci-test.sh`
+  * [pull-cluster-api-e2e-main] `./scripts/ci-e2e.sh`
+    * GINKGO_FOCUS: `[PR-Blocking]`
+* optional for merge, run if go code changes:
+  * [pull-cluster-api-apidiff-main] `./scripts/ci-apidiff.sh`
+  * [pull-cluster-api-e2e-informing-main] `./scripts/ci-e2e.sh`
+    * GINKGO_FOCUS: `[PR-Informing]`
+    * GINKGO_SKIP: `[IPv6]`
+* optional for merge, run if manually triggered:
+  * [pull-cluster-api-test-mink8s-main] `./scripts/ci-test.sh`
+    * KUBEBUILDER_ENVTEST_KUBERNETES_VERSION: `1.24.2`
+  * [pull-cluster-api-e2e-mink8s-main] `./scripts/ci-e2e.sh`
+    * GINKGO_SKIP: `[Conformance] [K8s-Upgrade] [IPv6]`
+    * KUBERNETES_VERSION_MANAGEMENT: `stable-1.24`
+  * [pull-cluster-api-e2e-full-dualstack-ipv6-main] `./scripts/ci-e2e.sh`
+    * DOCKER_IN_DOCKER_IPV6_ENABLED: `true`
+    * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade]`
+  * [pull-cluster-api-e2e-ipv6-main] `./scripts/ci-e2e.sh`
+    * DOCKER_IN_DOCKER_IPV6_ENABLED: `true`
+    * GINKGO_FOCUS: `[IPv6] [PR-Informing]`
+    * IP_FAMILY: `IPv6`
+  * [pull-cluster-api-e2e-full-main] `./scripts/ci-e2e.sh`
+    * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade] [IPv6]` (i.e. "no tags")
```

Comment:
```suggestion
    * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade]|[IPv6]`
```
I think no tags is something that depends on which tags we have overall in our e2e tests, so this might not be true (or stay true)